Read the number of cores and maximum memory to use from environment instead of hard coding
author: Christoph Siegert (Leipzig University)

diff -ruN RoseTTAFold.orig/run_e2e_ver.sh RoseTTAFold/run_e2e_ver.sh
--- RoseTTAFold.orig/run_e2e_ver.sh	2021-08-16 12:54:04.081235616 +0200
+++ RoseTTAFold/run_e2e_ver.sh	2021-08-16 13:06:25.547251482 +0200
@@ -15,8 +15,8 @@
 SCRIPT=`realpath -s $0`
 export PIPEDIR=`dirname $SCRIPT`
 
-CPU="8"  # number of CPUs to use
-MEM="64" # max memory (in GB)
+CPU=${CPU:-"8"}  # number of CPUs to use
+MEM=${MEM:-"64"} # max memory (in GB)
 
 # Inputs:
 IN="$1"                # input.fasta
diff -ruN RoseTTAFold.orig/run_pyrosetta_ver.sh RoseTTAFold/run_pyrosetta_ver.sh
--- RoseTTAFold.orig/run_pyrosetta_ver.sh	2021-08-16 12:54:04.081235616 +0200
+++ RoseTTAFold/run_pyrosetta_ver.sh	2021-08-16 13:07:02.619152279 +0200
@@ -15,8 +15,8 @@
 SCRIPT=`realpath -s $0`
 export PIPEDIR=`dirname $SCRIPT`
 
-CPU="8"  # number of CPUs to use
-MEM="64" # max memory (in GB)
+CPU=${CPU:-"8"}  # number of CPUs to use
+MEM=${MEM:-"64"} # max memory (in GB)
 
 # Inputs:
 IN="$1"                # input.fasta
